packages/frontend/src/components/project/KnowledgeNuggets.tsx

```diff
@@ -0,0 +1,63 @@
+import { KnowledgeNugget } from '@l2beat/config'
+import React from 'react'
+
+import { ArrowRightIcon } from '../icons/Arrows'
+import { OutLink } from '../OutLink'
+
+export interface KnowledgeNuggetsProps {
+  knowledgeNuggets?: KnowledgeNugget[]
+}
+
+export function KnowledgeNuggets(props: KnowledgeNuggetsProps) {
+  const { knowledgeNuggets } = props
+
+  if (!knowledgeNuggets) return null
+
+  return (
+    <div className="KnowledgeNuggets px-4 md:px-0">
+      <span className="text-[28px] leading-[32.81px] font-bold">
+        Knowledge nuggets
+      </span>
+      <div className="flex flex-col gap-3 p-4 mt-4">
+        {knowledgeNuggets.map((nugget) => (
+          <KnowledgeNuggetItem knowledgeNugget={nugget} key={nugget.title} />
+        ))}
+      </div>
+    </div>
+  )
+}
+
+interface KnowledgeNuggetProps {
+  knowledgeNugget: KnowledgeNugget
+}
+
+function KnowledgeNuggetItem(props: KnowledgeNuggetProps) {
```

Comment:
As we have a single file per component everywhere (or almost everywhere) else please extract this too. There can still be just a single story for the list of knowledge nuggets.

Reply:
Okay, sure :)